airflow/models/taskinstance.py

```diff
@@ -343,7 +344,7 @@ class TaskInstance(Base, LoggingMixin):
     task_id = Column(String(ID_LEN, **COLLATION_ARGS), primary_key=True, nullable=False)
     dag_id = Column(String(ID_LEN, **COLLATION_ARGS), primary_key=True, nullable=False)
     run_id = Column(String(ID_LEN, **COLLATION_ARGS), primary_key=True, nullable=False)
-    map_index = Column(Integer, primary_key=True, nullable=False, default=-1)
+    map_index = Column(Integer, primary_key=True, nullable=False, server_default=text("-1"))
```

Comment:
This one can possibly have default and server_default.

Reply:
I checked some sources and realised `default` isn’t actually much useful in ORM, only if we use the SQL expression builder. And with `server_default` set, `default` only really matters for `UPDATE`, which actually seems a bit wrong to me, so I’ll omit `default` until they are proven useful.